textdb/textdb-sandbox/team3datafile.txt

```diff
@@ -0,0 +1,4 @@
+networkx 1.1.1
```

Comment:
@sandeepreddy602 : what's a good folder structure for teams to store their own data files?